src/main/plugins/discordrpc.ts

```diff
@@ -102,7 +102,15 @@ export default class DiscordRichPresence {
             return;
         }
         const listenURL = `https://cider.sh/p?s&id=${attributes.playParams.id}` // cider://play/s/[id] (for song)
-        const AMWebURL = `https://music.apple.com/us/song/${attributes.playParams.catalogId}`
+        let AMWebURL = '';
+        if(attributes.playParams.catalogId  == null){
+            AMWebURL = `https://music.apple.com/${attributes.storefrontId}/song/${attributes.playParams.id}`
```

Comment:
seems unnecessary for the conditional except the console.log statement

Reply:
if catalogID is null then use id